src/commands/appSettings/AzureWebJobsStoragePromptStep.ts

```diff
@@ -45,7 +45,9 @@ export class AzureWebJobsStoragePromptStep<T extends IAzureWebJobsStorageWizardC
     }
 
     public shouldPrompt(context: T): boolean {
-        return !context.azureWebJobsStorageType && context.storageAccount === undefined && context.newStorageAccountName === undefined;
+        //Only want to prompt if no storage account it selected
```

Comment:
Was this written by afternoon Megan? lmao
```suggestion
        // Only should prompt if no storage account was selected
```

Reply:
It was but also included distractions from book talk which doesn't help :)